src/avfoundation.cs

```diff
@@ -3172,6 +3172,110 @@ interface AVCaptureSynchronizedDepthData
 		AVCaptureOutputDataDroppedReason DroppedReason { get; }
 	}
 
+	[TV (11,0), NoWatch, Mac (10,13), iOS (11,0)]
+	[Protocol, Model]
```

Comment:
The documentation isn't very clear about how this is used, so IMHO it would be best to remove `[Model]` (and `[BaseType]`) for now.